config/routes.rb

```diff
@@ -9,5 +9,5 @@
 
 
   # Root path to enter the react application
-  root 'survey#index'
+  root 'surveys#index'
```

Comment:
One thing that can be an issue with rails it you need to keep track of the naming scheme from some things, plurality can be important.